### DISCUSSION TITLE
IPXE RTL8153

### DISCUSSION THREAD

**Discussion by filisdiez** [ipxe/ipxe#257] [General]
Hello!

I want to download the latest ipxe from usb flash drive for this there is a usb network card TP-Link UE300 v1 with chip rtl8153. Compiled IPXE with the make bin/ecm.usb EMBED = scripts/default.ipxe option, when loading from a usb flash drive, my menu appears and even the network one is determined, but when I download some image writes connecting and throws it out by timeout. What else can I do to get it loaded?

[url=https://radikal.ru][img]https://b.radikal.ru/b16/2102/c8/bea158990176.jpg[/img][/url]

**Comment by filisdiez** [comment]
https://b.radikal.ru/b16/2102/c8/bea158990176.jpg

**Reply by mcb30** [reply]
Your screenshot appears to show everything working correctly: you have obtained an address via DHCP and you have chosen to enter the iPXE interactive shell.

What were you trying to do?

**Reply by filisdiez** [reply]
I checked again from local ip everything loads, but from the domain name does not load and from white ip also does not want to reset by timeout. It feels like there's no Internet on the network card.

**Reply by filisdiez** [reply]
I download it like this http://mysite.ru:28051/Win10/wimboot

**Reply by mcb30** [reply]
> I checked again from local ip everything loads, but from the domain name does not load and from white ip also does not want to reset by timeout. It feels like there's no Internet on the network card.

I'm afraid your problem report is not very useful, because you are guessing what might be happening instead of just telling us exactly what you see.  Please read https://www.chiark.greenend.org.uk/~sgtatham/bugs.html for some advice on how to report problems.

**Reply by filisdiez** [reply]
https://c.radikal.ru/c29/2102/00/1f883c56f977.png

**Reply by filisdiez** [reply]
https://b.radikal.ru/b20/2102/0a/021aafdf70ba.png
 sorry

**Reply by mcb30** [reply]
Your image:

![image](https://b.radikal.ru/b20/2102/0a/021aafdf70ba.png)

shows the error message "Connection timed out (http://ipxe.org/4c0a6035)".

If you follow that link, you'll see that the error is a TCP timeout.  This shows that DNS has succeeded in resolving the host name, and iPXE is attempting to connect to that host, but is getting no response.

Check the output of the [`route`](https://ipxe.org/cmd/route) command to make sure that your default gateway is configured correctly.

Also check that the server (and intervening network) will allow connections on the custom port 37862 that you are trying to use.  Most web servers on the Internet will not allow connections from non-standard port numbers.
